### ISSUE TITLE
Activator method doesn't do anything

### ISSUE THREAD

**Issue by machineonamission** [lgariv/appsort#1]
I downloaded this tweak and setup the Activator action, but whenever i do the action, nothing happens. I've tried multiple different actions and respringing, but it just doesn't do anything. Inside Activator it shows that those actions are binded.
iPhone 11 pro running unc0ver on ios 13.5

`oslog | grep AppSort` isn't outputting anything, even when i try to use the activation method

Also Activator does work, I tested shaking the device to lock the phone and that works, but the assignment to AppSort doesnt.

**Comment by jedimasterluis** [comment]
I’m having the same exact issue. Activator also works

**Comment by lgariv** [comment]
@HexCodeFFF @jedimasterluis 

I tried, but couldn’t reproduce the problem. More testing needed. 

Just to make sure; when doing the activator method, while on the home screen, you get no alert pop up? 
I have set my action to volume up and volume down, this is the alert you should see:
![image](https://user-images.githubusercontent.com/5081484/85545174-5dfa8b00-b624-11ea-9795-4c7afe2c0561.png)

**Comment by machineonamission** [comment]
Nope, I’m not getting that pop up. I also changed my activator method to volume up and it doesn’t fix it. I also tried reinstalling activator and AppSort.

**Comment by jedimasterluis** [comment]
If it helps, mine worked on version before first release but not no more

**Comment by lgariv** [comment]
> If it helps, mine worked on version before first release but not no more

Hm. That's not supposed to happen. I did not change anything regarding Activator. Thanks for letting me know!
Can you try again the version before this and see if it works?

**Comment by jedimasterluis** [comment]
![E77A5EC3-DF43-435D-BC88-158FB79146A5](https://user-images.githubusercontent.com/10966745/85955742-a58b5a80-b935-11ea-9f65-9061cb64c162.png)
![Uploading 49DD0F3D-8AE2-40EC-B31D-064FD908A446.png…]()

**Comment by jedimasterluis** [comment]
This is previous version

**Comment by machineonamission** [comment]
Downgrading the version makes the menu pop up, but tapping the Hue button crashes springboard

**Comment by RokuD1** [comment]
I’m experiencing this as well, main issue I found is when Appsort assigns to activator, activator views the assignment as (unknown) and doesn’t do anything.

**Comment by chelaxian** [comment]
Hello to all. Same here. Install lua 5.3.5-1 and liblua 5.3.5-1 than install this AppSort deb from Filza. Tweak is shown up in settings and let me choose Activator action to set up. In activators bindings I see “unknown” action and nothing happens when trigger this action. XS Max 13.5

**Comment by chelaxian** [comment]
2nd beta has working activators action (shown up in Activators bindings fine - as “App Sort” instead of “unknown”) and action shows pop up window with sorting Options. But choosing any options crashes springboard. No lua and liblua were needed to install this beta

**Comment by chelaxian** [comment]
Can you make tweak without activator? For example - put sorting options in settings tab of tweak instead of pop up
